hibernate-core/src/main/java/org/hibernate/internal/util/xml/XMLHelper.java

```diff
@@ -59,6 +59,14 @@ public DocumentFactory getDocumentFactory() {
 
 	public SAXReader createSAXReader(ErrorLogger errorLogger, EntityResolver entityResolver) {
 		SAXReader saxReader = new SAXReader();
+		try {
+			saxReader.setFeature( "http://apache.org/xml/features/nonvalidating/load-external-dtd", false );
```

Comment:
Should we add a comment, perhaps copying description from [Xerces doc](http://xerces.apache.org/xerces-j/features.html)?
Could be something like:
`saxReader.setFeature( "http://apache.org/xml/features/nonvalidating/load-external-dtd", false );  // Ignore the external DTD completely
`